### PR TITLE
Deploy v2

### PR DESCRIPTION
**Pull request by sogipec**

### CODE REVIEW

File: test/hardhat/distributionCreator/distributionCreator.test.ts [hunk 1]
```diff
@@ -447,7 +469,7 @@ contract('DistributionCreator', () => {
         propToken1: 2000,
         propFees: 4000,
         isOutOfRangeIncentivized: 0,
-        epochStart: startTime,
+        epochStart: startTime + 1000,
```
Comment (GuillaumeNervoXS) [inline test/hardhat/distributionCreator/distributionCreator.test.ts:472]: why did we have to increase this?

File: test/hardhat/middleman/merklFraxIncentivizationHandler.test.ts [hunk 2]
```diff
@@ -230,64 +230,70 @@ contract('MerklFraxIncentivizerHandler', () => {
       await middleman
         .connect(alice)
         .incentivizePool(alice.address, ZERO_ADDRESS, ZERO_ADDRESS, angle.address, 0, parseEther('0.7'));
-      expect(await manager.nonces(middleman.address)).to.be.equal(1);
-      expect(await angle.balanceOf(manager.address)).to.be.equal(parseEther('0'));
-      expect(await angle.balanceOf(bob.address)).to.be.equal(parseEther('0.7'));
+      const time = await latestTime();
+      expect(await angle.balanceOf(manager.address)).to.be.equal(parseEther('0.07'));
```
Comment (GuillaumeNervoXS) [inline test/hardhat/middleman/merklFraxIncentivizationHandler.test.ts:234]: why did this amount changed?

Comment (0xtekgrinder) [reply test/hardhat/middleman/merklFraxIncentivizationHandler.test.ts:234]: agEUR no longer has a fee reduction

File: test/hardhat/distributionCreator/distributionCreator.test.ts (hunk 1 above)
Comment (0xtekgrinder) [reply test/hardhat/distributionCreator/distributionCreator.test.ts:472]: because else the startTime is in the past compared to the current timestamp

**Review by GuillaumeNervoXS — APPROVED** [review]

**Pull request merged**